Sources/Public/iOS/FilepathImageProvider.swift

```diff
@@ -18,16 +18,20 @@ public class FilepathImageProvider: AnimationImageProvider {
   ///
   /// - Parameter filepath: The absolute filepath containing the images.
   ///
-  public init(filepath: String) {
+  public init(filepath: String, contentsGravity: CALayerContentsGravity = .resize) {
     self.filepath = URL(fileURLWithPath: filepath)
+    self.contentsGravity = contentsGravity
   }
 
-  public init(filepath: URL) {
+  public init(filepath: URL, contentsGravity: CALayerContentsGravity = .resize) {
     self.filepath = filepath
+    self.contentsGravity = contentsGravity
   }
 
   // MARK: Public
 
+  public var contentsGravity: CALayerContentsGravity
```

Comment:
This is a class so we probably want this to be internal and immutable

```suggestion
  let contentsGravity: CALayerContentsGravity
```